google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/StorageImpl.java

```diff
@@ -84,6 +84,10 @@
   private static final String EMPTY_BYTE_ARRAY_MD5 = "1B2M2Y8AsgTpgAmY7PhCfg==";
   private static final String EMPTY_BYTE_ARRAY_CRC32C = "AAAAAA==";
   private static final String PATH_DELIMITER = "/";
+  /**
+   * SignedUrls uses GCS XML API endpoint.
```

Comment:
Use: `Signed URLs are only supported through the GCS XML API endpoint.`

Reply:
resolved